common/views/components/FeaturedCard/FeaturedCard.js

```diff
@@ -167,7 +167,7 @@ const FeaturedCardExhibitionBody = ({
   exhibition,
 }: FeaturedCardExhibitionBodyProps) => {
   return (
-    <>
+    <div data-test-id="featured-exhibition">
```

Comment:
Is this card used for anything else? Why is it `FeaturedCard`?

Reply:
I'm not sure I understand the question. The `FeaturedCard` can take various `children`. The `FeaturedCardExhibition` takes this.

It's the thing that looks like this at the top of `/whats-on`

![image](https://user-images.githubusercontent.com/1394592/98565011-fe381000-22a4-11eb-8167-36d561a57284.png)

Reply:
What I am concerned about is that we are adding a `featured-exhibition` `id` to the `FeaturedCard`, which, from what I understand can be a featured anything?

Reply:
We're adding it to the `FeaturedCardExhibitionBody`, which only exists on FeaturedCards that are exhibitions.

Reply:
Got it - my fault.